specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/storage.json

```diff
@@ -3794,7 +3795,8 @@
           "type": "string",
           "enum": [
             "Enabled",
-            "Disabled"
+            "Disabled",
+            "SecuredByPerimeter"
```

Comment:
Please update the description of "publicNetworkAccess" properties for SecuredByPerimeter.